ci/actions-templates/linux-builds-template.yaml

```diff
@@ -55,6 +56,8 @@ jobs:
             snap_arch: i386  # skip-pr skip-master
           - target: aarch64-unknown-linux-gnu # skip-pr
             snap_arch: arm64 # skip-pr
+          - target: aarch64-unknown-linux-musl # skip-pr
+            snap_arch: arm64 # skip-pr
```

Comment:
Is `aarch64-unknown-linux-musl` really a valid snapcraft architecture?  I don't think so.  This hunk should not be present.

Reply:
Done, fixed in 6184ff0